packages-next/auth/src/gql/getMagicAuthLinkSchema.ts

```diff
@@ -82,7 +82,7 @@ export function getMagicAuthLinkSchema({
             // Save the token and related info back to the item
             const { token, itemId } = result;
             await itemAPI.updateOne({
-              id: itemId,
+              id: itemId.toString(),
```

Comment:
If this is to satisfy the types, I'm not sure how I feel about it, but stylistically I'd prefer

```
id: `${itemId}`,
```

Which functionally has the same effect (including ensuring the type will be `string`)